pdns/dnspacket.cc

```diff
@@ -360,12 +360,6 @@ void DNSPacket::wrapup()
     addTSIG(pw, d_trc, d_tsigkeyname, d_tsigsecret, d_tsigprevious, d_tsigtimersonly);
   
   d_rawpacket.assign((char*)&packet[0], packet.size()); // XXX we could do this natively on a vector..
-
-  // copy RR counts so LPE can read them
```

Comment:
I wonder if leaving this in would avoid surprises at a later time. It does not look like it hurts anyway.